app/views/content/landing/how-to-become-a-teacher/_content.html.erb

```diff
@@ -28,7 +28,7 @@
       </p>
 
       <p>
-        If you're interested in teaching certain subjects, you might be able to get a scholarship or bursary of up to <%= v :scholarships_generic_maxshortened %> tax-free.
+        If you're interested in <%= link_to("teaching certain subjects", page_path("life-as-a-teacher/explore-subjects")) %>, you might be able to get a scholarship or bursary of up to <%= v :scholarships_generic_maxshortened %> tax-free.
```

Comment:
I'm not sure this is a good link to include - we will be building out the explore subjects section to include non-bursary subjects?

This is also a paid search/social landing page so is noindexed - so not so relevant for SEO. Plus we tried to have a principle of only one signposting link per section on these pages!